core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -408,17 +409,25 @@ struct SchedulerCountMetrics {
 }
 
 impl SchedulerCountMetrics {
-    fn maybe_report_and_reset(&mut self, should_report: bool) {
+    fn maybe_report_and_reset(
+        &mut self,
+        should_report: bool,
+        container: &TransactionStateContainer,
+    ) {
         const REPORT_INTERVAL_MS: u64 = 1000;
         if self.interval.should_update(REPORT_INTERVAL_MS) {
             if should_report {
-                self.report();
+                self.report(container);
             }
             self.reset();
         }
     }
 
-    fn report(&self) {
+    fn report(&self, container: &TransactionStateContainer) {
+        let (min_prioritization_fees, max_prioritization_fees) = container
+            .get_min_max_prioritization_fees()
+            .into_option()
+            .unwrap_or_default();
```

Comment:
this is the wrong place to call this. This would get the minimum and maximum at the time of reporting!
We want the overall min/max we've seen in the entire reporting period. Just call it in the main loop and reduce min/max there.

Reply:
Ok

Reply:
Idea was to get minimum and maximum values when we start a new slot. With new design there is no way to say if the new slot was started so I did it while reporting. Doing it in main loop adds additional question when should we reset ?

Reply:
I think it makes sense to just reset it during reporting like we do with the other metrics. That way we are always seeing the min/max observed during each 1s period.

Reply:
But then it will be the same thing as it is right now or am I missing something ?

Reply:
Ok agreed and fixed.